File: hackerrank/usAndUkPart2/usAndUkPart2.py
def main():
	N = input()
	lines = []
	for i in range(N):
		line = raw_input()
		lines.append(line)

	T = input()

	for i in range(T):
		UKword = raw_input()
		count = 0


		USWord = UKword.replace("our", "or")

		for i in range(N):
			words = lines[i].split(" ")
			for word in words:
				if UKword == word:
					count = count + 1
				elif USWord == word:
					count = count + 1
		print(count)

main()

File: hackerrank/usAndUkPart2/usAndUkPart2.cpp
#include <iostream>
#include <cstring>

using namespace std;

int main(){
	int N;
	cin >> N;

	vector<list <string>> lines(N);

	for(int i = 0; i < N; i++){
		string word;
		cin >> word;
		lines[i].push_back(word);
	}
}